docs/troubleshooting/clean-install.md

```diff
@@ -0,0 +1,128 @@
+# How to clean install Streamlit
+
+If you run into any issues while installing Streamlit or if you just want to do a clean install, don't worry, we've got you covered. In this guide, we'll show you how to clean install Streamlit for Windows, macOS, and Linux.
+
+```eval_rst
+.. contents::
+    :local:
+    :depth: 1
+```
+
+## Install Streamlit on Windows
```

Comment:
I would push back a little bit and ask a few follow up questions: 

* Do you expect your customers to explicitly search for this topic using the OS? Not from on site nav but from Google. Since these articles are similar, they may compete with one another for search rank/traffic. 
* I'm not sure if your theme supports "tabs", but we may want to consider a single page with a tab per OS. Then you consolidate the traffic to a single page, but remove the messiness of all content in one long page. 

With that said, why don't we deploy the page as is, see what type of traffic and issues come up, then make adjustments?

Reply:
The tabs idea is great, but I don't think it's supported by default. And it would be awesome if the OS choice was remembered across pages and sessions. This would be doable, but we'd probably have to implement it as a custom plugin.

As for SEO: I don't think we need to worry about it too much at this point. "Streamlit" is a unique enough word that even with no SEO strategy we're already at the top of the search results. If we end up showing up as 2 results (one for Windows and one for Mac/Linux) then I'm not sure there's really any loss for users.

Meanwhile, the reason I'd like to break this into 2 pages is because it already looks messy as is. Here's [a preview of the file](https://github.com/streamlit/streamlit/blob/305c93c4f3b7c226d35e38ab934ca06ecf149a1a/docs/troubleshooting/clean-install.md). As a single long page, I have a hard time finding the Mac/Linux section via the scrollbar. I also have a hard time scrolling back to the top of that section after I reached the bottom.

So I think it makes sense to prioritize the doc-reading experience over SEO optimizations at this point.

Either way, I'll merge this as is right now, and will break into pages when I do the release.